src/main/java/org/fenixedu/academic/domain/student/personaldata/EducationLevelType.java

```diff
@@ -0,0 +1,61 @@
+package org.fenixedu.academic.domain.student.personaldata;
+
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import org.fenixedu.academic.domain.exceptions.DomainException;
+import org.fenixedu.academic.util.Bundle;
+import org.fenixedu.bennu.core.domain.Bennu;
+import org.fenixedu.bennu.core.i18n.BundleUtil;
+import org.fenixedu.commons.i18n.LocalizedString;
+
+public class EducationLevelType extends EducationLevelType_Base {
+    protected EducationLevelType() {
+        super();
+        setRootDomainObject(Bennu.getInstance());
+    }
+
+    public static EducationLevelType create(String code, LocalizedString name, boolean active, boolean isForStudent,
```

Comment:
Lets add a more friendly create method, just with the code, name and active parameters. And this one we'll use just for the initial creation of the entities from the enum vales.